src/silx/math/colormap.py

```diff
@@ -489,18 +503,18 @@ def apply_colormap(
 
 class NormalizeResult(NamedTuple):
     data: numpy.ndarray
-    vmin: numbers.Number
-    vmax: numbers.Number
+    vmin: float
+    vmax: float
```

Comment:
Note that now the python base types are supposed to support `int < float`

The other problem could be with types like `numpy.float32`, but in this case we should use `numbers.Number` everywhere instead of `float`, or maybe `mypy` will just allow that at some point, if it's not already the case.